docs/sources/features/datasources/stackdriver.md

```diff
@@ -174,6 +178,19 @@ Example Alias By: `{{resource.type}} - {{metric.type}}`
 
 Example Result: `gce_instance - compute.googleapis.com/instance/cpu/usage_time`
 
+### SLO Queries
+
+{{< docs-imagebox img="/img/docs/v70/slo-query-builder.png" max-width= "400px" class="docs-image--right" >}}
+
+To get an understanding of the basic concepts in service monitoring, please refer to Google Stackdriver's [official docs](https://cloud.google.com/monitoring/service-monitoring).
+
+The SLO query builder in the Stackdriver data source allows you to display SLO data in time series format. Begin by choosing a `Project`. Then select a `Service` and a `SLO`. Use the `Selector` field to specify which time series selector to use.
```

Comment:
Same as above, one needs to select `Query Type` first.

Reply:
👍